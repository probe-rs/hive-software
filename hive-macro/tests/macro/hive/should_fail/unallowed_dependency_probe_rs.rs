#[hive_macro::hive]
pub mod tests {
    #![allow(unused)]
    use hive_test;
    use probe_rs::Session;
    use std;
    use {
        std::sync,
        std::time,
        {std::alloc, std::io},
    };
}

fn main() {}
