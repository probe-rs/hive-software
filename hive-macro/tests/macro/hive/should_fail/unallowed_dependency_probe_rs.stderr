error: The dependency 'probe_rs' is not allowed in Hive test modules
 --> tests/macro/hive/should_fail/unallowed_dependency_probe_rs.rs:5:9
  |
5 |     use probe_rs::Session;
  |         ^^^^^^^^
  |
  = help: Use the alias 'probe_rs_test' instead
  = note: example: use probe_rs_test::Session;
